client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+    <div>
+        <b-alert
+            v-if="errorMessage"
+            class="mt-2"
+            :show="dismissCountDown"
+            variant="info"
+            @dismissed="dismissCountDown = 0">
+            {{ errorMessage }}
+        </b-alert>
+        <b-row align-v="center">
+            <b-col v-if="display == 'radio'">
+                <b-form-group>
+                    <!-- Select single from radio -->
+                    <b-form-radio-group v-model="currentValue" :options="formattedOptions" text-field="label" stacked>
+                    </b-form-radio-group>
+                </b-form-group>
+            </b-col>
+            <b-col v-else-if="display == 'checkboxes'">
+                <!-- Select multiple from checkboxes -->
+                <b-form-group>
+                    <b-form-checkbox-group
+                        v-model="currentValue"
+                        :options="formattedOptions"
+                        value-field="value"
+                        text-field="label"
+                        stacked>
+                    </b-form-checkbox-group>
+                </b-form-group>
+            </b-col>
+            <b-col v-else>
+                <!-- Multiple select from drop down -->
+                <multiselect
+                    v-if="multiple"
+                    v-model="currentValue"
+                    :options="formattedOptions"
+                    :multiple="true"
+                    placeholder="Select value"
+                    deselect-label=""
+                    select-label=""
+                    track-by="value"
+                    label="label">
+                </multiselect>
+                <!-- Single select from drop down -->
+                <multiselect
+                    v-else
+                    v-model="currentValue"
+                    :options="formattedOptions"
+                    deselect-label=""
+                    select-label=""
+                    label="label"
+                    track-by="value">
+                    {{ currentValue }}
+                </multiselect>
+            </b-col>
+        </b-row>
+    </div>
+</template>
+
+<script>
+import Multiselect from "vue-multiselect";
+
+export default {
+    components: {
+        Multiselect,
+    },
+    props: {
+        value: {
+            default: null,
+        },
+        defaultValue: {
+            type: [String, Array],
+            required: false,
+            default: null,
```

Comment:
Default `required` is `false`, so no need to explicitly define both. Defining either `required: true` or `default: ?` is sufficient.